rally2-matching-system/tests/test-routine.sh

```diff
@@ -2,10 +2,19 @@
 
 echo -e "\nRally Matching System Test Routine Started..."
 
+# Set the mode, required
+
+mode=$1
+
+if [ -z $mode ]; then
```

Comment:
Should check for valid values.
If someone were to try to run the old command, this parameter would be filled by the host.